wled00/set.cpp

```diff
@@ -19,21 +19,52 @@ void handleSettingsSet(AsyncWebServerRequest *request, byte subPage)
   //WIFI SETTINGS
   if (subPage == SUBPAGE_WIFI)
   {
-    char oldSSID[sizeof(clientSSID)];
-
-    strcpy(oldSSID, clientSSID);
-    strlcpy(clientSSID,request->arg(F("CS")).c_str(), 33);
-    if (!strcmp(oldSSID, clientSSID)) forceReconnect = true;
+    unsigned cnt = 0;
+    for (size_t n = 0; n < WLED_MAX_WIFI_COUNT; n++) {
+      char cs[4] = "CS"; cs[2] = 48+n; cs[3] = 0; //client SSID
+      char pw[4] = "PW"; pw[2] = 48+n; pw[3] = 0; //client password
+      char ip[5] = "IP"; ip[2] = 48+n; ip[4] = 0; //IP address
+      char gw[5] = "GW"; gw[2] = 48+n; gw[4] = 0; //GW address
+      char sn[5] = "SN"; sn[2] = 48+n; sn[4] = 0; //subnet mask
+      if (request->hasArg(cs)) {
+        if (n >= multiWiFi.size()) multiWiFi.push_back(WiFiConfig()); // expand vector by one
+        char oldSSID[33]; strcpy(oldSSID, multiWiFi[n].clientSSID);
+        char oldPass[65]; strcpy(oldPass, multiWiFi[n].clientPass);
+
+        strlcpy(multiWiFi[n].clientSSID, request->arg(cs).c_str(), 33);
+        if (strlen(oldSSID) == 0 || !strncmp(multiWiFi[n].clientSSID, oldSSID, 32)) {
+          forceReconnect = true;
```

Comment:
Seems that setting the SSID to empty while connected to the WiFi doesn't triggers this ForceReconnect.

It's a silly issue because after a restart it'll behavior as intended, but I wonder how deep would be this issue